apps/dashboard/src/pages/workflows.tsx

```diff
@@ -33,14 +31,13 @@ export const WorkflowsPage = () => {
         }
       >
         <OptInModal />
-        <div className="mt-3 flex justify-between px-2.5 py-2">
-          <div className="invisible flex w-[20ch] items-center gap-2 rounded-lg bg-neutral-50 p-2">
-            <RiSearch2Line className="text-foreground-400 size-5" />
-            <Input placeholder="Search workflows" />
-          </div>
+        <div className="flex justify-between px-2.5 py-2.5">
+          <div className="invisible flex w-[20ch] items-center gap-2 rounded-lg bg-neutral-50 p-2"></div>
 
           <CreateWorkflowButton asChild>
-            <Button variant="primary">Create workflow</Button>
+            <Button variant="primary" size="sm">
+              Create workflow
+            </Button>
```

Comment:
The page header was not aligned as per the designs. This was quite noticeable between other page transitions from integration store